homework2/fastq_filtrator.py

```diff
@@ -0,0 +1,75 @@
+#checking GC content according to conditions
+def check_gc (seq, gc_bound_min, gc_bound_max):
+    gc_content = 100 * (sum(char == "G" or char == "C" for char in seq)) / len(seq)
+    if gc_content >= gc_bound_min and gc_content <= gc_bound_max:
+        return True
+    return False
+
+#checking lenght according to conditions
+def check_lenght (seq, length_bound_min, length_bound_max):

```

Comment:
Этот пробел не нужен
```suggestion
def check_lenght(seq, length_bound_min, length_bound_max):
```